docs/setup/self-hosted/docker.mdx

```diff
@@ -49,35 +18,54 @@ It is the Docker image of MindsDB that comes with [these integrations](https://g
 It is the Docker image of MindsDB that comes with [these integrations](https://github.com/mindsdb/mindsdb/blob/staging/default_handlers.txt) and the Hugging Face integration preloaded.
 </Tip>
 
-Run the command below, providing the selected Docker image, to start MindsDB in Docker and follow the logs:
+## Prerequisites
+
+Before proceeding, ensure you have installed Docker, following the [official Docker documentation](https://docs.docker.com/install).
+
+## Setup
+
+This setup of MindsDB uses one of the available Docker images, as listed above.
+
+Follow the steps to set up MindsDB in a Docker container.
+
+### Install MindsDB
+
+Run this command to start MindsDB in Docker:
 
 ```bash
-docker run -p 47334:47334 -p 47335:47335 mindsdb/mindsdb
+docker run --name mindsdb_container -p 47334:47334 -p 47335:47335 mindsdb/mindsdb
 ```
 
-<Info>
-Alternatively, if you run this command with the `detach` flag, it will return the container ID and not follow the logs:
+Where:
+
+-   `docker run` is a native Docker command used to spin up a container.
+-   `--name mindsdb_container` defines a name for the container.
+-   `-p 47334:47334` publishes port 47334 to access MindsDB GUI and HTTP API.
+-   `-p 47335:47335` publishes port 47335 to access MindsDB MySQL API.
+-   `mindsdb/mindsdb` is a Docker image provided by MindsDB. You can choose a different one from the list above.
+
+Once the container is created, you can use the following commands:
+
+-   `docker stop mindsdb_container` to stop the container. Note that this may not always be necessary because when turning off the host machine, the container will also be shut down.
+-   `docker start mindsdb_container` to start the container back up.
```

Comment:
I think it might be important to mention here that this command is used to restart a stopped container with all its previous changes (such as any dependencies that are installed) intact.
I am not sure if it is necessary to mention that using the above `docker run` again when the previous container is stopped will start an entirely new container. What do you think?

Reply:
Good point. I added more explanation to distinguish between docker start and docker run.